scripts/development/01-setup-minikube.sh

```diff
@@ -13,7 +13,8 @@ minikube start \
   --memory=16g \
   --disk-size=100g \
   --driver=hyperkit \
-  --wait=true
+  --wait=true \
+  --kubernetes-version=1.21.12
```

Comment:
@steved I'm a little confused by this because I'm able to resolve DNS entries for existing buildkit pods. Everything works as expected with the following:

- minikube – `v1.25.2`
- k8s version – `v1.23.3`
- ingress-dns image – `gcr.io/k8s-minikube/minikube-ingress-dns:0.0.2@sha256:4abe27f9fc03fedab1d655e2020e6b165faf3bf6de1088ce6cf215a75b78f05f`

It looks like it was fixed 8 months ago via https://github.com/kubernetes/minikube/commit/803fc43e264005ddd750c128fed56306b9b2a941#diff-d7a36f96adfbf699458fe0f8015812c97d63deb017534219147fbdb1dd0e19cdR483 where they started using their own image instead of the cryptex one, right?

I'm curious what your setup is and what behavior you observed.

Reply:
Same exact versions... and this time it worked. DNS is finicky. I've removed that commit.

Reply:
weird.